docs-src/concepts/what-is-a-timer.md

```diff
@@ -0,0 +1,25 @@
+---
+id: what-is-a-timer
+title: What is a Timer?
+sidebar_label: Timer
+description: A Workflow can set a durable timer for a fixed time.
+tags:
+  - term
+  - explanation
+---
+
+You can add a delay to your Workflow Execution by setting a Timer.
```

Comment:
Currently, I think this sounds a lot like "Start Delay".

I think we either we want to be generically accurate and say something like "SDKs offer Timer APIs so that Workflow Executions are deterministic in their handling of time values".

Or.. say something highly contextualized like
"You can sleep inside a Workflow Execution for seconds, minutes, or years."
And then break it down from there.